pages/tutorials/[slug].js

```diff
@@ -30,10 +33,70 @@ export async function getStaticProps(context) {
 }
 
 export default function Tutorial({ name, steps, id, slug }) {
+  const [currentStep, setCurrentStep] = useState(0);
+
   return (
     <>
       <Headline>{name}</Headline>
-      <TutorialStartCard />
+      {currentStep === 0 ? (
+        <>
+          <TutorialStartCard />
+          <ButtonContainer>
+            <Button
+              isPrimary={true}
+              onClick={() =>
+                setCurrentStep((prevCurrentStep) => prevCurrentStep + 1)
+              }
+            >
+              start
+            </Button>
+          </ButtonContainer>
+        </>
+      ) : currentStep < steps.length ? (
+        <>
+          <TutorialCard step={steps[currentStep]} />
+          <ButtonContainer>
+            <Button
+              isPrimary={false}
+              onClick={() =>
+                setCurrentStep((prevCurrentStep) =>
+                  prevCurrentStep > 0 ? prevCurrentStep - 1 : 0
+                )
+              }
+            >
+              prev
+            </Button>
+            <Button
+              isPrimary={true}
+              onClick={() =>
+                setCurrentStep((prevCurrentStep) =>
+                  prevCurrentStep < steps.length
+                    ? prevCurrentStep + 1
+                    : prevCurrentStep
+                )
+              }
+            >
+              next
+            </Button>
+          </ButtonContainer>
+        </>
+      ) : (
+        <>
+          <h2>done</h2>
+          <ButtonContainer>
+            <Button
+              isPrimary={false}
+              onClick={() =>
+                setCurrentStep((prevCurrentStep) =>
+                  prevCurrentStep > 0 ? prevCurrentStep - 1 : 0
+                )
+              }
+            >
+              prev
```

Comment:
Since this feature has already been requested several times (and since it is so easy to implement with my button component 😉 ), I will be happy to fulfill this demand for you.